src/components/PVSearchBar.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { SearchBar } from 'react-native-elements'
+import { useGlobal } from 'reactn'
+
+type Props = {
+  containerStyle?: any
+  inputContainerStyle?: any
+  onChangeText: any
+  onClear: any
+  placeholder?: string
+  value?: string
+}
+
+export const PVSearchBar = (props: Props) => {
+  const { containerStyle, inputContainerStyle, onChangeText, onClear, placeholder, value } = props
+  const [globalTheme] = useGlobal('globalTheme')
+
+  return (
+    <SearchBar
+      clearIcon={true}
+      containerStyle={[containerStyle, globalTheme.textInputWrapper]}
+      inputContainerStyle={[inputContainerStyle, globalTheme.textInput]}
```

Comment:
This and the above prop should first use the global theme and then the prop. Order matters when passing same style properties